### PR TITLE
Chainrunner tweaks

### PR DESCRIPTION
**Pull request by h4nsu**
# Description

Small tweaks in Python chainrunner:
- make raw chainspec an option when boostraping the chain
- allow passing lists of values in set_flags
- improve on-the-fly log verbosity changing
- add address method to node
- make run_nodes.py use the same config flags as run_nodes.sh

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: local-tests/run_nodes.py [hunk 1]
```diff
@@ -7,39 +7,46 @@
...
+print(f'Bootstrapping chain for {nodes} nodes')
+chain.bootstrap(binary,
+                keys[:nodes],
                 chain_type='local')
 chain.set_flags('validator',
-                port=Seq(30334),
-                ws_port=Seq(9944),
-                rpc_port=Seq(9933),
+                'unsafe-ws-external',
+                'unsafe-rpc-external',
```
Comment (pmikolajczyk41) [inline local-tests/run_nodes.py:32]: maybe also `'enable-log-reloading'`

**Review by pmikolajczyk41 — APPROVED** [review]

File: local-tests/chainrunner/chain.py [hunk 2]
```diff
@@ -62,7 +65,12 @@ def _set_flags(nodes, *args, **kwargs):
                 n.flags[k] = True
         for k, v in kwargs.items():
             for i, n in enumerate(nodes):
-                n.flags[k] = v + i if isinstance(v, Seq) else v
+                if isinstance(v, Seq):
+                    n.flags[k] = v + i
+                elif isinstance(v, list) and len(v) == len(nodes):
```
Comment (kostekIV) [inline local-tests/chainrunner/chain.py:70]: should be added to documentation of the `set_flags...` functions

**Review by kostekIV — APPROVED** [review]
lgtm

File: local-tests/chainrunner/chain.py [hunk 3]
```diff
@@ -48,6 +50,7 @@ def bootstrap(self, binary, validators, nonvalidators=None, **kwargs):
         def account_to_node(account):
             n = Node(binary, chainspec, op.join(self.path, account), self.path)
             n.flags['node-key-file'] = op.join(self.path, account, 'p2p_secret')
+            n.flags['enable-log-reloading'] = True
```
[the remark this answers is not in the document]
Comment (h4nsu) [reply local-tests/chainrunner/chain.py:53]: It seems to be working without that flag

File: local-tests/chainrunner/chain.py (hunk 2 above)
Comment (h4nsu) [reply local-tests/chainrunner/chain.py:70]: Done

**Pull request merged**